Panic when a variable is not defined

Panic when a variable is not defined:
```
2020-05-15 21:37:25 INFO: validating environment variables in REDACTED.yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x18bcbff]

goroutine 1 [running]:
github.com/Praqma/helmsman/internal/app.substituteVarsInYaml(0xc000112500, 0x71, 0xc00025fbb0, 0x0)
	/go/src/github.com/Praqma/helmsman/internal/app/utils.go:194 +0x30f
github.com/Praqma/helmsman/internal/app.substituteVarsInStaticFiles(0xc0003acf00)
	/go/src/github.com/Praqma/helmsman/internal/app/utils.go:152 +0x78c
github.com/Praqma/helmsman/internal/app.fromYAML(0x7ffeefbff439, 0x8, 0xc0003acf00, 0x2, 0x2, 0x1)
	/go/src/github.com/Praqma/helmsman/internal/app/utils.go:118 +0x147
github.com/Praqma/helmsman/internal/app.(*state).fromFile(0xc0003acf00, 0x7ffeefbff439, 0x8, 0x0, 0x0, 0xc0003b2c30)
	/go/src/github.com/Praqma/helmsman/internal/app/state.go:53 +0x11a
github.com/Praqma/helmsman/internal/app.(*cli).readState(0x23ff040, 0xc0003acdc0)
	/go/src/github.com/Praqma/helmsman/internal/app/cli.go:224 +0x49b
github.com/Praqma/helmsman/internal/app.Main()
	/go/src/github.com/Praqma/helmsman/internal/app/main.go:36 +0xba
main.main()
	/go/src/github.com/Praqma/helmsman/cmd/helmsman/main.go:8 +0x20
```
Helmsman 3.4.1, Darwin build

--- a/internal/app/logging.go
+++ b/internal/app/logging.go
@@ -9,6 +9,18 @@ import (
 
 type Logger struct {
 	*logger.Logger
+	LoggerInterface
+}
+
+type LoggerInterface interface {
+	Info(string)
+	Debug(string)
+	Verbose(string)
+	Error(string)
+	Warning(string)
+	Notice(string)
+	Critical(string)
+	Fatal(string)
 }
 
 var log *Logger
@@ -43,6 +55,13 @@ func (l *Logger) Warning(message string) {
 
 func (l *Logger) Notice(message string) {
 	baseLogger.Notice(message)
+}
+
+func (l *Logger) Critical(message string) {
+	if _, err := url.ParseRequestURI(settings.SlackWebhook); err == nil {
+		notifySlack(message, settings.SlackWebhook, true, flags.apply)
+	}
+	baseLogger.Critical(message)
 }
 
 func (l *Logger) Fatal(message string) {

